src/components/boardPage/EditUserModal/EditUserModal.tsx

```diff
@@ -0,0 +1,109 @@
+import React, { useEffect, useState } from "react";
+import * as ReactDOM from "react-dom";
+import "./EditUserModal.scss";
+import { Modal } from "../../common/Modal/Modal";
+import { useModalState } from "../../../hooks/useModalState";
+import { useUserState } from "../../../hooks/useUserState";
+
+interface formData {
+  firstName: string;
+  lastName: string;
+}
+
+export const EditUserModal = () => {
+  const { modalsVisibility, changeEditUserModalVisibility } = useModalState();
+  const { user, patchUserData } = useUserState();
+  const { id, firstName, lastName } = user.userData || {};
```

Comment:
before line 14, you may place the following

```
const { userData = {} } = user
```

It is an alternative, not necessary; since it is a matter of style. 

Consider the following trade-off: Adding one line but increasing code readability.